CONTRIBUTING.md

```diff
@@ -72,6 +66,17 @@ Please ensure your commits pass code quality assurance by executing:
 [keyword args to the constructor]: http://pyqt.sourceforge.net/Docs/PyQt5/qt_properties.html
 
 
+Human Interface Guidelines
+--------------------------
+For UI design, conform to the [OS X Human Interface Guidelines].
+In a nutshell, use title case for titles, push buttons, menu titles
+and menu options. Elsewhere, use sentence case. Use title case for
+combo box options where the item is imperative (e.g. Initialize with Method)
+and sentence case otherwise.
+
+[OS X Human Interface Guidelines]: (https://developer.apple.com/library/mac/documentation/UserExperience/Conceptual/OSXHIGuidelines/TerminologyWording.html).
```

Comment:
In this format, the URL is unparenthesized (to make a link). Also no trailing comma.